DevOps-local/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3.7'
```

Comment:
DevOps-local > environment

Reply:
Also, you can add some small instruction into `README.md` file. 

## Local usage...

### Prerequisites 

1. Docker
2. Docker compose

In order to run locally... 

`cd environment && docker compose up -d `